i18n/locales/en.json

```
@@ -26,6 +26,8 @@
   "エラーの詳細を見る": "Click to view error details",
   "クリックして閉じる": "Click to close",
   "コマンド": "Command",
+  "検証": "Verify",
+  "自動検証(Experimental)": "Auto verify(Experimental)",
```

```suggestion
  "自動検証(Experimental)": "Auto verify (Experimental)",
```
英語では括弧の外側にスペースがある形が好まれます。